template/build/webpack.base.conf.js

```diff
@@ -20,8 +21,7 @@ module.exports = {
     filename: '[name].js'
   },
   resolve: {
-    extensions: ['', '.js', '.vue', '.json'],
-    fallback: [path.join(__dirname, '../node_modules')],
```

Comment:
Can't find webpack2 equivalent `resolve.modules` in your config:

https://webpack.js.org/configuration/resolve/#resolve-modules